### DISCUSSION TITLE
Allow dynamic sorting of sites on different fields

### DISCUSSION THREAD

**Discussion by santo74** [umami-software/umami#1523] [General]
Right now, the websites on the dashboard are always statically sorted, more specifically in alphabetical order.
But as others already pointed out it would be nice if they could be sorted based on dynamic data such as number of visitors.
Actually it would be desirable to be able to sort them on the fly on fields such as views, visitors, bounce rate, average visit time and of course site name (the current behaviour).

[This issue](https://github.com/umami-software/umami/issues/1169) is related, but while it states at the end it is "Fixed in v1.37.0.", this is still only predefined, static sorting.

In addition it would also be nice if there was some kind of "compact mode" because right now I can only see 2 websites before I have to start scrolling (and this is already with the charts collapsed).
This makes it really hard when having tens of websites, especially when the most visited ones are at the bottom of the list.

**Comment by mikecao** [comment]
The problem is for people who have many websites, like 100s, then you have to run the stats for each website to determine the totals for sorting.

**Reply by santo74** [reply]
If you have 100s of websites and you're afraid of the load implications then you simply don't use the dynamic sorting. 
But imho, if you have that many websites, you should divide them over multiple accounts/dashboards anyway.
And if you mention the implications of dynamically sorting +100 websites in the docs and maybe even add a hint in the web UI, then I don't see this as a major problem tbh.

Also, you can (and should) store totals of older data so that you don't have to recalculate them each time.
